cmd/sync-rover-groups/main.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	ldapv3 "github.com/go-ldap/ldap/v3"
+	"github.com/sirupsen/logrus"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/test-infra/prow/flagutil"
+	"k8s.io/test-infra/prow/logrusutil"
+	"sigs.k8s.io/yaml"
+
+	templatev1 "github.com/openshift/api/template/v1"
+	userv1 "github.com/openshift/api/user/v1"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+type options struct {
+	logLevel         string
+	manifestDirRaw   flagutil.Strings
```

Comment:
if it's a set then it should be called `manifestDirsRaw` (plural)

Reply:
I missed this one. Strange.
On it now.

Reply:
https://github.com/kubernetes/test-infra/blob/681f3531eceb7bc827d6f99de567cd9087bcbcfc/prow/flagutil/github.go#L44

Maybe singular is also fine?